### ISSUE TITLE
Extremely large dependency trees cause a memory failure

### ISSUE THREAD

**Issue by evanweible-wf** [kevmoo/pubviz#15]
For packages with very large dependency trees, the file is generated but the page cannot be loaded. Instead, this error is shown:

```
abort("Cannot enlarge memory arrays. Either (1) compile with  -s TOTAL_MEMORY=X  with X higher than the current value 16777216, (2) compile with  -s ALLOW_MEMORY_GROWTH=1  which adjusts the size at runtime but prevents some optimizations, (3) set Module.TOTAL_MEMORY to a higher value before the program runs, or if you want malloc to return NULL (0) instead of this abort, compile with  -s ABORTING_MALLOC=0 ") at Error
    at Ga (https://kevmoo.github.io/pubviz/viz.js:49:124)
    at Fa (https://kevmoo.github.io/pubviz/viz.js:49:22)
    at v (https://kevmoo.github.io/pubviz/viz.js:1172:101)
    at Function.ba [as Wa] (https://kevmoo.github.io/pubviz/viz.js:34:488)
    at vc (https://kevmoo.github.io/pubviz/viz.js:1093:402)
    at M4 (https://kevmoo.github.io/pubviz/viz.js:1157:7144)
    at bK (https://kevmoo.github.io/pubviz/viz.js:1148:184533)
    at aK (https://kevmoo.github.io/pubviz/viz.js:1148:184286)
    at Rg (https://kevmoo.github.io/pubviz/viz.js:1147:242859)
    at yi (https://kevmoo.github.io/pubviz/viz.js:1153:66518)
If this abort() is unexpected, build with -s ASSERTIONS=1 which can give more information.
```

**Comment by kevmoo** [comment]
Yup – this is the limit of the emscriptem compiled code. I'd love to know how many nodes and edges your graph has...

**Comment by evanweible-wf** [comment]
@kevmoo I sent you an email with the pubviz output

**Comment by kevmoo** [comment]
Just published 2.3.10 w/ this fix. 😄

**Comment by evanweible-wf** [comment]
Awesome! Thanks for the quick turnaround @kevmoo
